app/code/Magento/ReviewGraphQl/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "magento/module-review-graph-ql",
+    "description": "N/A",
+    "type": "magento2-module",
+    "require": {
+        "php": "~7.1.3||~7.2.0||~7.3.0",
+        "magento/module-catalog": "*",
+        "magento/module-review": "*",
+        "magento/framework": "*"
+    },
+    "suggest": {
+        "magento/module-graph-ql": "*",
```

Comment:
I'm not sure here, but I assume that we may require the `magento/module-graph-ql` module not just suggest